src/components/Auth/models.js

```diff
@@ -0,0 +1,50 @@
+const { Schema } = require('mongoose');
+const mongooseConnection = require('../../config/connection');
+
+const AuthUserSchema = new Schema(
+    {
+        email: {
+            type: String,
+            unique: true,
+            required: true,
+        },
+        passwordHash: {
+            type: String,
+            required: true,
+            select: false,
+        },
+    },
+    {
+        collection: 'auth_users',
+        versionKey: false,
+    },
+);
+const AuthUserModel = mongooseConnection.model('AuthUserModel', AuthUserSchema);
+
+const RefreshTokenSchema = new Schema(
```

Comment:
better separate between files 
like:
...Auth/models/users.js
...Auth/models/tokens.js
or 
...Auth/schemas/user.js
...Auth/schemas/token.js

Reply:
thought about this, tnx! 
also, maybe you have a link to a good article on how to do migrations in mongo / mongoose?